app/controllers/concerns/foreman/controller/registration_commands.rb

```diff
@@ -19,6 +19,14 @@ def insecure
     registration_params['insecure'] ? '--insecure' : ''
   end
 
+  def is_token_valid(min_value, max_value)
+    registration_params['jwt_expiration'].to_i.between?(min_value, max_value)
+  end
+
+  def unlimited
+    registration_params['jwt_expiration'].to_i == 0 || registration_params['jwt_expiration'] == 'unlimited'
```

Comment:
We are  accepting 'unlimited' because it's coming from the UI. The following case `'zz'.to_i" will not occur because there is no possibility of string input.